lib/cache/cache.go

```diff
@@ -350,10 +350,6 @@ type Cache struct {
 	// fails.
 	initErr error
 
-	// wrapper is a wrapper around cache backend that
-	// allows to set backend into failure mode,
-	// intercepting all calls and returning errors instead
-	wrapper *backend.Wrapper
```

Comment:
Looks like this wrapper is only being used in cache_test.go now. Is it still needed or can it be removed completely?